scripts/format-toml.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/nixos-23.11.tar.gz
+#! nix-shell -i bash --pure
+#! nix-shell -p bash taplo
```

Comment:
Could this be `nix shell nixpkgs#bash nixpkgs#taplo` instead?

Reply:
I can simplify this a bit but these get treated as Nix expressions rather than CLI arguments to be resolved so you get syntax errors with `nixpkgs#taplo`. You can do `"pkgs.taplo"` but that means the same thing anyway